vector/src/test/java/im/vector/app/features/settings/devices/v2/DevicesViewModelTest.kt

```diff
@@ -164,20 +181,24 @@ class DevicesViewModelTest {
     private fun givenDeviceFullInfoList(): List<DeviceFullInfo> {
         val verifiedCryptoDeviceInfo = mockk<CryptoDeviceInfo>()
         every { verifiedCryptoDeviceInfo.isVerified } returns true
```

Comment:
Maybe we should remove the `isVerified` mocks since we use `trustLevel` instead?

Reply:
Right, done.